### PR TITLE
Splitters improvements

### PR DESCRIPTION
**Pull request by feldlime**
- Renamed `KFoldSplitter` -> `RandomSplitter`
- Renamed argument `test_size` -> `test_fold_frac` for `RandomSplitter`
- Changed logic and initialization of `LastNSplitter` in case of `n_splits > 1`
- Changed initialization of `TimeRangeSplitter` (instead of `date_range` argument, use `test_size` and `n_splits`)

### CODE REVIEW

**Comment by blondered** [comment]
Can we add `i_split` (or `i_fold`) to fold_info for all splitters? It simplifies the code for user

File: rectools/model_selection/time_split.py [hunk 1]
```diff
@@ -25,25 +25,25 @@
...
-        Left will be included, right will be excluded from fold.
-        Interactions before first border will be used for train.
-        Interaction after right border will not be used.
-        Ca be easily generated with [`pd.date_range`]
-        (https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.date_range.html)
+    test_size : str
+        Size of test fold in format ``[1-9]\d*[DH]``, e.g. 1D, 4H.
+        Test folds are taken from the end of `interactions`.
+        The last fold includes full time unit with the last interaction.
+        E.g. if last interaction was in 01:25 a.m. of Mondat, then
```
[the remark this answers is not in the document]
Comment (feldlime) [reply rectools/model_selection/time_split.py:42]: fixed

File: rectools/model_selection/last_n_split.py [hunk 2]
```diff
@@ -50,47 +52,41 @@ class LastNSplitter(Splitter):
...
     >>> interactions = Interactions(df)
     >>>
-    >>> lns = LastNSplitter(2, False, False, False)
-    >>> for train_ids, test_ids, _ in lns.split(interactions):
+    >>> splitter = LastNSplitter(2, 2, False, False, False)
+    >>> for train_ids, test_ids, _ in splitter.split(interactions):
     ...     print(train_ids, test_ids)
-    [0 2 4 5] [1 3 6 7 8]
+    [0 1 2 5] [3 4 6 7]
+    [0] [1 2 5]
```
[the remark this answers is not in the document]
Comment (feldlime) [reply rectools/model_selection/last_n_split.py:70]: agreed, changed

File: tests/model_selection/test_last_n_split.py [hunk 3]
```diff
@@ -45,168 +45,104 @@ def _shuffle(values: tp.Sequence[int]) -> tp.List[int]:
...
+        actual = list(splitter.split(interactions, collect_fold_stats=True))
         pd.testing.assert_frame_equal(interactions.df, interactions_copy.df)
 
         assert len(actual) == 2
         assert len(actual[0]) == 3
+        assert len(actual[1]) == 3
 
-        assert sorted(actual[0][0]) == norm([0, 1, 2, 4, 5, 6])
-        assert sorted(actual[0][1]) == norm([3, 7, 8])
+        assert sorted(actual[0][0]) == norm([0, 1, 2, 5])
```
[the remark this answers is not in the document]
Comment (feldlime) [reply tests/model_selection/test_last_n_split.py:72]: done (used `to_shuffled`)

**Review by blondered — APPROVED** [review]

**Pull request merged**